elasticsearch/sgconfig/roles.yml

```diff
@@ -36,6 +36,7 @@ kibana_server:
 sg_role_prometheus:
   cluster:
     - METRICS
+    - SEARCH #required for index level metrics
```

Comment:
I would consider adding this permission group to `METRICS` instead https://github.com/openshift/origin-aggregated-logging/blob/ea15554f30d5fcff6806d33af222e95aea9c85d4/elasticsearch/sgconfig/action_groups.yml#L98